sda-commons-server-auth/src/main/java/org/sdase/commons/server/auth/key/OpenIdProviderDiscoveryKeySource.java

```diff
@@ -88,6 +92,10 @@ public String toString() {
     return "OpenIdProviderDiscoveryKeySource{" + "issuerUrl='" + issuerUrl + '\'' + '}';
   }
 
+  public String getIssuerUrl() {
+    return issuerUrl;
+  }
```

Comment:
As said below, I think it is not necessary to validate domains of the issuer in the token compared to the issuer of the configuration because we compare the exact required issuer. So it is not needed to expose the value here.

Maybe logging a warning if requiredIssuer and issuerUrl do not match the same domain on startup could be an alternative.

Reply:
This way sounds better and I would prefer this way to go.